kolibri/plugins/coach/assets/src/views/coach-exercise-render-page/attempt-log-list.vue

```diff
@@ -1,44 +1,41 @@
 <template>
 
-  <div class="answer-history">
+  <div class="attempt-log-list">
```

Comment:
@indirectlylit I wanted to know your opinion on this. I used the terminology of the models because it made it easier for me to reason about the views. I'm going to be mapping the model values for the state; do you think I should be renaming these to be more apparent in the frontend?